src/utils/preprocessing.py

```diff
@@ -416,3 +426,77 @@ def posenet_processing(filename):
     image = cv2.resize(image, (401, 401))
 
     return image
+
+
+def blur_faces_in_file(source_path: str, target_path: str) -> bool:
```

Comment:
You moved the functionality. Should we also move the unit tests?

Reply:
yes, I will add tests too

Reply:
Please make sure to add the pytest command to `pytest-az-pipeline.yaml`: This currently only does flake8 although it is has pytest in the name

Reply:
@gangooteli

Reply:
it is there in dockerfile.Build file

Reply:
@markus-hinsche 
check this https://dev.azure.com/cgmorg/ChildGrowthMonitor/_build/results?buildId=1089&view=results